### PR TITLE
Vaurca Augments & Aut'akh Loadout Tweaks

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds two new augments for Vaurca, ~~shamelessly copying~~ based on existing Aut'akh augments - a stun weapon for security Warriors and a health analyser for medical.

Tweaks the cost of some species-locked augments to be more in line with the cheaper Aut'akh ones. Aut'akh augments are now culture restricted to Aut'akh now that it's an actual culture. 

The Vaurca combitool is now restricted to engineering and machinist, and is available to warriors in those roles. I wanted to replace it with a mechanical limb similar to the Aut'akh ones, but Bulwarks don't have mechanical limbs.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: code/modules/client/preference_setup/loadout/items/xeno/vaurca.dm [hunk 1]
```diff
@@ -240,14 +240,35 @@
 /datum/gear/augment/vaurcatool
 	display_name = "vaurca integrated toolset"
 	description = "A heavy arm-mounted toolset designed for use by Bound Workers and Bulwarks."
-	cost = 5
```
Comment (Arrow768) [inline code/modules/client/preference_setup/loadout/items/xeno/vaurca.dm:243]: Undocumented change. Revert that.

File: code/modules/client/preference_setup/loadout/items/augments.dm [hunk 2]
```diff
@@ -208,7 +208,7 @@
 	path = /obj/item/organ/internal/augment/tool/drill
 	whitelisted = list(SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_VAURCA_BULWARK, SPECIES_VAURCA_WORKER)
 	allowed_roles = list("Shaft Miner")
-	cost = 5
```
Comment (Arrow768) [inline code/modules/client/preference_setup/loadout/items/augments.dm:211]: Undocumented change. Revert it.

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/modules/organs/subtypes/vaurca.dm [hunk 3]
```diff
@@ -425,3 +425,108 @@
...
+				H.electrocute_act(10, owner, def_zone = target_zone)
+			else
+				H.stun_effect_act(0, 50, target_zone, owner)
+
+			owner.visible_message("<span class='danger'>[H] has been prodded with [src] by [owner]!</span>")
+			playsound(get_turf(owner), 'sound/weapons/Egloves.ogg', 50, 1, -1)
+
+/obj/item/organ/external/hand/right/vaurca/medical
+	name = "medical grasper"
+	action_button_name = "ACtivate Integrated Biological Analyser"
```
Comment (NonQueueingMatt) [inline code/modules/organs/subtypes/vaurca.dm:492]:
```suggestion
	action_button_name = "Activate Integrated Biological Analyser"
```

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**